README.md

```diff
@@ -215,6 +215,7 @@ Releasing a new version is quite simple, as the packages are build and distribut
     - Once the tag with the new Rasa SDK release is pushed and the package appears on [pypi](https://pypi.org/project/rasa-sdk/), the dependency in the rasa repository can be resolved (see below).
 2. Switch to the branch you want to cut the release from (`master` in case of a major / minor, the current feature branch for patch releases) 
     - Update the `rasa-sdk` entry in `pyproject.toml` with the new release version and run `poetry update`. This creates a new `poetry.lock` file with all dependencies resolved.
+    - Commit the changes with `git commit -am "bump rasa-sdk dependency"` but do not push them. They will be automatically picked up by the following step.
```

Comment:
I don't get this one. There needs to be an updated `poetry.lock`. Where is this handled in the release script?

Reply:
the step above tells you to update the dependency and then run `poetry update`. But at this point in time, your git directory is "dirty", and running the next step `make release` fails because of that. So I added this step. Does it make sense?

Reply:
Sorry, for some reason I didn't see the lock file in the diff 👍